homeassistant/components/lightwave/sensor.py

```diff
@@ -45,6 +46,11 @@ def state(self):
         """Return the state of the sensor."""
         return self._state
 
+    @property
+    def state_class(self):
```

Comment:
We can set this via a class attribute as done for the device class:
https://github.com/home-assistant/core/blob/5bbf0ca6abf3f0117b61ee5162418bcd8b6e0ab0/homeassistant/components/lightwave/sensor.py#L28

Reply:
Change made & tested.